client/gefyra/misc/comps/service.py

```diff
@@ -23,7 +23,14 @@ def data(params: "GefyraInstallOptions") -> list[dict]:
             )
     if params.service_annotations:
         try:
-            stowaway_annotations.update(params.service_annotations)
+            for annotation in params.service_annotations:
+                try:
+                    # handle cli params as key=value
+                    key, value = annotation[0].split("=")
+                    stowaway_annotations[key] = value
+                except ValueError:
+                    # handle preset values
```

Comment:
We should probably display a warning or something if this happens and the parameter is ignored no?